apis/dscinitialization/v1/servicemesh_types.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+// ServiceMeshSpec configures Service Mesh.
+type ServiceMeshSpec struct {
+	// +kubebuilder:validation:Enum=Managed;Removed
+	// +kubebuilder:default=Removed
+	ManagementState operatorv1.ManagementState `json:"managementState,omitempty"`
+	// Mesh holds configuration of Service Mesh used by Opendatahub.
+	Mesh ControlPlaneSpec `json:"controlPlane,omitempty"`
```

Comment:
```suggestion
	Mesh ControlPlaneSpec `json:"mesh,omitempty"`
```

Reply:
@israel-hdez missing the field name update?

Reply:
Not sure why I missed the rename. Fixed.